packages/angular-workspace/example-client-app/src/app/customapp/customapp.component.html

```diff
@@ -311,12 +311,24 @@
                     Duration
                 </nimble-table-column-duration-text>
                 <nimble-table-column-text field-name="stringValue2" min-pixel-width="400" group-index="0">String 2</nimble-table-column-text>
+                <nimble-table-column-menu-button field-name="color" menu-slot="color-menu"
+                    (menu-button-column-beforetoggle)="onMenuButtonColumnBeforeToggle($event)"
+                >
+                    Color
+                </nimble-table-column-menu-button>
 
                 <nimble-menu slot="action-menu">
                     <nimble-menu-item>Item 1</nimble-menu-item>
                     <nimble-menu-item>Item 2</nimble-menu-item>
                     <nimble-menu-item>Item 3</nimble-menu-item>
                 </nimble-menu>
+
+                <nimble-menu slot="color-menu">
+                    <nimble-menu-item *ngFor="let color of possibleColors" (change)="onColorSelected(color)">
+                        <nimble-icon-check [style.visibility]="color === currentColor ? 'visible' : 'hidden'" slot="start"></nimble-icon-check>
```

Comment:
Note: I couldn't do an `ngIf` binding here like I wanted because of #2266. I'm working around the issue by binding to `style.visibility` instead.